src/ui/resource/loader.js

```diff
@@ -440,6 +440,11 @@ var Loader = Class(Emitter, function () {
           // Call the progress callback with isComplete == true
           cb && cb(src, failed, true, numLoaded, numResources);
 
+          // Emit progress event
+          this.emit('progress', {
+            progress: this.progress
+          });
```

Comment:
Yes, it just emits events on loader which inherits from Emitter. Just to be sure, I tested a build on android and the events came through.